inscriptions/ISic003418.xml

```diff
@@ -183,17 +180,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI178075">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Εἰρήνα νύμφη<lb n="2"/>ὧδε κεῖται.<lb n="3"/>κατὰ τοῦ μυστηρί-<lb n="4"/>ου οὖν τούτου μὴ τ-<lb n="5"/>ὶς ὧδε ἀνύξῃ.
-                    <lb n="6"/>
+                    <lb n="1"/>Εἰρῆνα Νύμφη
+                    <lb n="2"/>ὧδε κεῖται
+                    <lb n="3"/>κατὰ τοῦ μυστηρί
+                    <lb n="4" break="no"/>ου οὖν τούτου μη
+                    <lb n="5" break="no"/><choice><reg>δ</reg><orig>τ</orig></choice>ὶς ὧδε ἀνύξη
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after Orsi 1900</note>
+                    </app>
+                    <app loc="4-5">
+                        <note>μηδεὶς ἀνοίξη</note> <!-- should we keep the regularised form here and add in the text with <choice> only the minor changes (I have always tried to leave the text as it appears on the stone by inserting <choice>etc. in case of normalisations/errors)? does this create problems for future lemmatisation? -->
```

Comment:
We need to have a discussion about apparatus fairly soon, to decide what to do with both encoding and display. We can use the <reg> to generate the app entry and the <orig> to present the unedited form in the text. Same logic for <sic> and <corr>. It's a point in the discussion doc, I think.

Reply:
Argh, I've missed the quotation marks for the code. I was saying that we can use `<reg>` and `<orig>` to generate the original and regularised form - one in the text, the other in the apparatus. Same for `<sic>` and `<corr>`.